src/cdi/CDI.hh

```diff
@@ -201,7 +201,7 @@ static double Planck2Rosseland(double const freq, double const exp_freq) {
   double factor;
```

Comment:
Please add a Doxygen comment block for this function.  Start with this template and update:
```
//---------------------------------------------------------------------------//
/*!
 * \brief Compute the difference between an integrated Planck and Rosseland
 * curves over \f$ (0,\nu) \f$.
 *
 * This helper function is used by CDI::integrate_planck_rosseland (also in this file).
 * 
 * ==> explain why you made the approximations that were made.
 * ==> explain why you want to use std::expm1
 * 
 * \param  freq The frequency for the upper limit of the integrand. 
 * \param  exp_freq exp(-freq)
 * \return The difference between the integrated Planck and Rosseland 
 * curves over \f$ (0,\nu) \f$.
 */
```